app/src/interfaces/m2a-builder/m2a-builder.vue

```diff
@@ -576,9 +576,9 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-.m2a-builder {
-	background-color: white;
-}
+// .m2a-builder {
+// background-color: white;
+// }
 
```

Comment:
Lets remove it instead of comment it out

```suggestion
```

Reply:
I didn't remove it because I was skeptical a change so small wouldn't come with issues, so I left it. 😁
But if you think we can remove it, here we go 🚀